src/services/theming/theming.svc.coffee

```diff
@@ -82,7 +82,9 @@ angular
       # configuring dashboard settings
       dhbSettings:
         inWidgetsContainer: false
-        showSyncApps: -> true
```

Comment:
@ouranos @alexnoox 
Warning: please note that somewhere as it will have to be changed next time you update impac-angular in your projects